.github/dependabot.yml

```diff
@@ -40,14 +40,6 @@ updates:
     timezone: America/Chicago
   open-pull-requests-limit: 99
   rebase-strategy: disabled
-- package-ecosystem: npm
-  directory: "/grader_host"
```

Comment:
Do we need to add `/apps/grader-host` to dependabot?

Reply:
Nope - because Dependabot supports Yarn workspaces, everything should Just Work.

Reply:
Nice!